build/commands/lib/config.js

```diff
@@ -1331,4 +1333,11 @@ Object.defineProperty(Config.prototype, 'outputDir', {
   },
 })
 
+Object.defineProperty(Config.prototype, 'useSiso', {
+  get: function () {
+    // Android fails with multiple reasons currently. Disable it for now.
+    return this.useRemoteExec && this.hostOS === 'linux' && !this.isAndroid()
```

Comment:
`siso` is actually a full replacement for `ninja`, but the way we configure it now is to enable it for remote execution only. This means public builds are still on `ninja`. The switch for public builds will be separate from this PR.